src/components/CustomModal.tsx

```diff
@@ -1,36 +1,13 @@
 import React, { ReactNode } from 'react'
-import { Modal } from 'react-native'
```

Comment:
Obviously not part of your PR but I think it would make sense to rename `CustomModal`, at least I don't now what the `custom` stands for and where it is used.

Reply:
Well i think there are cases where it make sense to have sth like "CustomModal" or "LModal" for Lunes, since sometimes it's annoying when you have ambiguous imports (Modal React Native or other core libs)and accidentally take the wrong one. Sure you can do an *as import. Just to tell you why i renamed it

Reply:
I think our current approach most of the time is to name react native and library imports `as RNMyLibaryComponent`. I think this really makes sense.